src/Sylius/Bundle/AddressingBundle/Doctrine/ORM/ZoneRepository.php

```diff
@@ -1,36 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sylius package.
- *
- * (c) Paweł Jędrzejewski
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sylius\Bundle\AddressingBundle\Doctrine\ORM;
-
-use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
-use Sylius\Component\Addressing\Repository\ZoneRepositoryInterface;
-
-/**
- * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
- */
-class ZoneRepository extends EntityRepository implements ZoneRepositoryInterface
-{
-    /**
-     * {@inheritdoc}
-     */
-    public function findByName($name)
-    {
-        return $this->createQueryBuilder('o')
-            ->addSelect('members')
-            ->leftJoin('o.members', 'members')
-            ->where('o.name = :name')
-            ->setParameter('name', $name)
-            ->getQuery()
-            ->getResult()
-        ;
-    }
-}
```

Comment:
This method is only used in a Behat context and I'm not sure you will ever use it in your application, because it's more likely that you will use `code` for querying. 

Using "default" joins is a separate topic, if we want to introduce it we should do it for all entities.